spec.html

```diff
@@ -39885,6 +39929,18 @@ <h1>ValueOfReadEvent( _execution_, _R_ )</h1>
         1. Return ComposeWriteEventBytes(_execution_, _R_.[[ByteIndex]], _Ws_).
       </emu-alg>
     </emu-clause>
+
+    <emu-clause id="sec-agentssynchronizewith" aoid="AgentsSynchronizeWith">
+      <h1>AgentsSynchronizeWith( _execution_ )</h1>
```

Comment:
Having both Agent*s*SynchronizeWith and AgentSynchronize*s*With is confusing to me.

In conjunction with my above comment about making [[AgentSynchronizesWith]] a List, I recommend inline the logic in this AO directly into the synchronizes-with section, should be a 1-liner.

Reply:
done